src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -97,7 +101,7 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
 
       <div className="room-modal__main">
         {room.image_url ? (
-          <img src={room.image_url} alt={room.title} />
+          <div className="room-modal__icon" style={iconStyle}></div>
         ) : (
           <span>{room.title}</span>
         )}
```

Comment:
I think the `span` here will have styling issues/look strange if it ever gets rendered. You could check this by hacking the code to render that code path (eg. making the check `!room.image_url` temporarily) to see what it looks like. Might be worth including a screenshot of that here as well to help with the review.

I know it's existing code, but it's probably irrelevant to render the `room.title` here again since it's already being rendered just above.

We may just want to always render the `room-modal__icon` `div`, and have it fall back to the `background-color` when it has an invalid URL (which is how the CSS works by default)

Reply:
Looks like this with the span
<img width="496" alt="Screen Shot 2021-03-25 at 6 20 36 PM" src="https://user-images.githubusercontent.com/51083763/112523583-d13d5300-8d96-11eb-97aa-f36aa5ca3d8f.png">

Like this with no image but just the div (better imo)
<img width="475" alt="Screen Shot 2021-03-25 at 6 21 55 PM" src="https://user-images.githubusercontent.com/51083763/112523783-0649a580-8d97-11eb-9c60-6fa10c3b6b22.png">

Reply:
Agreed. Let’s just keep the ‘div’ for both cases and drop the ‘spam’ entirely.